authors/apps/authentication/tests/test_views.py

```diff
@@ -115,3 +118,28 @@ def test_update_a_registered_user_after_register(self):
                               **headers, format='json')
         self.assertEqual(rev.status_code, 200)
 
+    def test_reset_password(self):
+        self.client.post('/api/users/', self.user, format = 'json')
+        response = self.client.post('/api/users/password_reset/', self.valid_email, format='json')
+        self.assertEqual(response.status_code, 201)
+
+    def test_reset_password_missing_email(self):
+        self.client.post('/api/users/', self.user, format = 'json')
+        response = self.client.post('/api/users/password_reset/', self.invalid_email, format='json')
+        self.assertEqual(response.status_code, 400)
+    
+    def test_change_password_missing_password(self):
+        self.invalid_password = {"new_password":""}
+        self.client.post('/api/users/', self.user, format = 'json')
+        self.res = self.client.post('/api/users/password_reset/', self.valid_email, format='json')
+        self.result = self.res.data['token']
+        response= self.client.put('/api/users/update_password/{}'.format(self.result), self.invalid_password, format='json')
```

Comment:
@emukungu , this line is too long

Reply:
refactored the code to fit pep8 standard